Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -643,7 +643,7 @@
     <value>Has other significant influence or control</value>
   </data>
   <data name="Supplier_ConnectedEntity_ConnectedEntityControlCondition_HasOtherSignificantInfluenceOrControlHint" xml:space="preserve">
-    <value>Has the right to exercise, or exercises, significant influence or control over the company. As defined in the &lt;a href="https://www.gov.uk/government/publications/guidance-to-the-people-with-significant-control-requirements-for-companies-and-limited-liability-partnerships" target="_blank"&gt;person of significant control (PSC) guidance&lt;/a&gt;.</value>
```

Comment:
This space before "guidance" was a `U+202F : NARROW NO-BREAK SPACE [NNBSP]` - I've made it a normal space